docs/site/angular-auth-oidc-client/docs/documentation/12-silent-renew.md

```diff
@@ -18,6 +18,39 @@ For session renewal with the PKCE flow, no iframes are used by default. Instead,
 
 Refresh tokens should be rotated and the refresh token should be revoked on a logout using the revocation endpoint.
 
+## Silent Renew Code Flow with PKCE using refresh tokens and no id_token returned in the refresh
+
+When an identity provider does not return an id_token in the refresh, the nonce cannot be validated in the id_token as it is not returned and needs to be deactivated. The following configuration should work for servers not returning an id_token in the refresh.
+
+```typescript
+import { NgModule } from '@angular/core';
+import { AuthModule, LogLevel } from 'angular-auth-oidc-client';
+
+@NgModule({
+  imports: [
+    AuthModule.forRoot({
+      config: {
+        authority: '--idp--',
+        redirectUrl: window.location.origin,
+        postLogoutRedirectUri: window.location.origin,
+        clientId: '--client_id--',
+        scope: 'openid profile offline_access',
+        responseType: 'code',
+        silentRenew: true,
+        useRefreshToken: true,
+        ignoreNonceAfterRefresh: true, // this is required if the id_token is not returned
+        triggerRefreshWhenIdTokenExpired: false, // required to refresh the browser if id_token is not updated after the first authentication
+        // allowUnsafeReuseRefreshToken: true, // this is required if the refresh token is not rotated
```

Comment:
Remove this commented out line

Reply:
this is for the docs, some IDPs require this.